blogs/5_how_to_gorilla.html

```diff
@@ -0,0 +1,502 @@
+                        <ul>
+                            <li> <a href="https://colab.research.google.com/drive/1I9UJoKh9sngE2MfPfQD5kbn2-twq2xvY?usp=sharing">Run OpenFunctions Locally!</a>
+                            </li> 
+                            <li> <a href="https://github.com/ShishirPatil/gorilla/tree/main/openfunctions#self-hosting-openfunctions">Self-Hosting OpenFunctions</a>
+                            </li>
+                        </ul>
+
+                    </p>
+
+                    <h4 id="integrate-third-party">Integrating Gorilla with Third Party Libraries 🦜</h4>
+                    <i> For a self contained walkthrough on how to integrate Gorilla with Langchain check out <a href="https://colab.research.google.com/drive/11HJWR3ylG1HSE2v78W1gRK-dKkSA0pHe">this Google Colab</a>. </i>
+                    <p>
+                        Integrate Gorilla with Langchain for easy deployment in less than 10 lines of code. 
+                        Install dependencies, create a langchain agent and start prompting!
+                        <ol>
+                            <li> Install dependencies:
+                                <br><code>pip install transformers[sentencepiece] datasets langchain openai==0.28.1 &> /dev/null</code>
+                            </li>
+                            <li>Define Langchain Chat Agent:
+                                <pre><code>
+                                    from langchain.chat_models import ChatOpenAI
+                                    chat_model = ChatOpenAI(
+                                        openai_api_base="http://zanino.millennium.berkeley.edu:8000/v1",
+                                        openai_api_key="EMPTY",
+                                        model="gorilla-7b-hf-v1",
+                                        verbose=True
+                                    )
+                                </code></pre>
+                            </li> 
+                            <li> Prompt:
+                                <br><code>example = chat_model.predict("I want to translate from English to Chinese")</code>
+                                <br><code>print(example)</code>
+                            </li>
+                        </ol>
+                    </p>
+
+                    <h4 id="api-zoo"> <a href="apizoo/">Gorilla API Zoo Index 📖</a></h4>
+                    <p>
+                        The API Zoo is an open sourced index containing API documentation that can be used by LLMs to increase tool-use capabilities via API calls. 
+                        Anyone can upload to the index, simply follow the instructions <a href="https://github.com/ShishirPatil/gorilla/blob/main/data/README.md#gorilla-api-store">here</a> to upload your API!
+                    </p>
+
+                    <h4 id="train-your-own-gorilla">Train Your Own Gorilla 🔨🔧</h4>
```

Comment:
This section needs a thorough review. 
1. An introduction paragraph on why you might want to train your own gorilla? What are the benefits?
2. What is needed (in terms of data formatting) to train your own Gorilla. 
3. What are the two ways you are suggesting below? What is the advantage of each?
4. Sub-headings for each of the 2 techniques. A description of what each of the technique represents.